performance/front/table-and-chart-with-good-performance.md

```diff
@@ -0,0 +1,41 @@
+# A table or a chart on my project takes to much time to render
+
+## Owner: [Justine Mignot](https://github.com/justinemignot)
+
+## Do you need to display all data you are displaying currently ?
+
+### Chart (Owner: [Thibaut Guedou](https://github.com/thibautguedou3))
```

Comment:
I would keep one owner in a MO, not have two, it can be Thibaut or you, but make it simpler for people to know directly who to ask.